pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -2092,6 +2092,9 @@ func rktRun(domainName, xenCfgFilename, imageHash string) (int, string, error) {
 		"--insecure-options=image",
 		"run",
 		imageHash,
+		//This is added to verify that setting env var works.
+		//TODO: Remove this once passing env var flow from UI is setup
+		"--set-env=FOO=bar",
```

Comment:
Can we make this real, rather than the stub? This will just last, and at some point months down the line, someone will say, "what is this doing here??"

If we have nowhere to source the env vars, then just make a simple key-value map that is unset with a comment to indicate where we will need to add it.

E.g.

```golang
// TODO: source these from image config
envVars := map[string]string{}
var (
    envVarSlice = make([]string,0)
)
for k, v := range envVars {
     envVarSlice = append(envVarSlice, fmt.Sprintf("--set-env=%s=%s", k, v))
}
// ...
// ...
	args := []string{
		"--dir=" + types.PersistRktDataDir,
		"--insecure-options=image",
		"run",
		imageHash,
		"--stage1-path=/usr/sbin/stage1-xen.aci",
		"--uuid-file-save=" + uuidFile,
	}
        args = append(args, envVarSlice...)


```

Reply:
Done.